tests/test_sourmash.py

```diff
@@ -72,6 +72,76 @@ def test_sourmash_info_verbose():
     assert "loaded from path" in err
 
 
+def test_load_pathlist_from_file_does_not_exist():
+    from sourmash.sourmash_args import load_pathlist_from_file
+    with pytest.raises(ValueError) as e:
+        load_pathlist_from_file("")
+    assert "file '' does not exist" in str(e.value)
+
```

Comment:
please add two lines between functions, thanks!

Reply:
Okay, I'll do that!